src/software/simulation/er_force_simulator.cpp

```diff
@@ -364,8 +365,15 @@ SSLSimulationProto::RobotControl ErForceSimulator::updateSimulatorRobots(
         auto& primitive_executor = primitive_executor_with_id.second;
         auto direct_control      = primitive_executor->stepPrimitive();
 
+        auto sim_state             = getSimulatorState();
+        const auto& sim_robots     = sim_state.blue_robots();
+        auto current_velocity_map  = getRobotIdToLocalVelocityMap(sim_robots);
+        auto ramped_direct_control = euclidean_four_wheel_convert.rampWheelVelocity(
+            current_velocity_map.at(robot_id), *direct_control,
+            primitive_executor_time_step);
```

Comment:
Could you add a comment to what this is doing and how it will affect the simulator